webrtc.html

```diff
@@ -11789,25 +11653,12 @@ <h3><code><dfn>RTCErrorDetailType</dfn></code> Enum</h3>
   "data-channel-failure",
   "dtls-failure",
   "fingerprint-failure",
-  "idp-bad-script-failure",
-  "idp-execution-failure",
-  "idp-load-failure",
-  "idp-need-login",
-  "idp-timeout",
-  "idp-tls-failure",
-  "idp-token-expired",
-  "idp-token-invalid",
   "sctp-failure",
   "sdp-syntax-error",
   "hardware-encoder-not-available",
   "hardware-encoder-error"
 };</pre>
         </div>
-        <p>
-          The error detail types whose names are prefixed with "idp-" are
-          used by the [[WEBRTC-IDENTITY]] specification. They are described
-          here because a WebIDL enum must be described in one place only.
```

Comment:
I suppose the identity spec will need to fake a partial enum here. @henbos am I misremembering or did you run into a similar problem recently? Did you find a good answer?

Reply:
I ended up making a `RTCErrorDetailTypeIdp` enum copy.